src/install.rst

```diff
@@ -6,361 +6,320 @@ Installing Nextstrain
+Steps vary by runtime option (Docker, native) and operating system (macOS, Windows, WSL on Windows, Linux).
+For help choosing, refer to our :doc:`/reference/faq`, such as:
 
-.. tabs::
-
-   .. group-tab:: macOS
-
-      .. note::
-
-         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for Augur <augur:installation/installation>` and :doc:`install Auspice via npm <auspice:introduction/install>`.
-
-      1. Install Miniconda:
-
-         .. The installer link is taken from https://docs.conda.io/en/latest/miniconda.html.
-
-         a. `Download the installer <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_.
-
-            .. note::
+  * :ref:`what-are-docker-conda-mamba-wsl-etc`
+  * :ref:`choosing-a-runtime`
+  * :ref:`when-to-use-wsl`
 
-                  This is the Intel x86 64-bit installer, :ref:`which we recommend even for Mac computers with Apple silicon (e.g. M1) <why-intel-miniconda-installer-on-apple-silicon>`.
+First, install a Nextstrain runtime.
 
-         b. Open the downloaded file and follow through installation prompts.
-
-      2. Open a terminal window.
-      3. Install Mamba on the ``base`` Conda environment:
-
-         .. code-block:: bash
-
-            conda install -n base -c conda-forge mamba --yes
-            conda activate base
-
-      4. Install the Nextstrain components. There are two options:
-
-         .. tabs::
+.. tabs::
 
-            .. group-tab:: Docker (recommended)
+   .. group-tab:: Docker
 
-               .. warning::
+      .. tabs::
 
-                  If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), **Native** installation is recommended due to slowness with the Docker installation. `We are considering ways to improve this <https://github.com/nextstrain/docker-base/issues/35>`_.
+         .. group-tab:: macOS
 
-               1. `Install Docker Desktop using the official guide <https://docs.docker.com/desktop/install/mac-install/>`_.
-               2. Create a Conda environment named ``nextstrain``:
+            .. warning::
 
-                  .. include:: snippets/conda-create-bash.rst
+               If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), the **native** runtime is recommended due to slowness with the Docker runtime. `We are considering ways to improve this <https://github.com/nextstrain/docker-base/issues/35>`_.
 
-               3. Install the Nextstrain CLI:
+            1. `Install Docker Desktop using the official guide <https://docs.docker.com/desktop/install/mac-install/>`_.
+            2. Install the Nextstrain CLI:
 
-                  .. code-block:: bash
+               .. code-block:: bash
 
-                     mamba install --yes nextstrain-cli
```

Comment:
Since the setup of a `nextstrain` Conda environment is no longer common across all runtimes, it'd make sense to also reword subsequent tutorials:

- 6b032ccb20cc1ad7551cfd3fe60438c6de8f0fa2 (add to this PR)
- https://github.com/nextstrain/ncov/commit/2016068a3436aaca5aec55b4c339c066fa5a04c8 (create external PR)

Reply:
Ah, yes! I added the first to this PR. Will you open an external PR for the second since it's your change?

Reply:
Opened: https://github.com/nextstrain/ncov/pull/1002